cli/flox/src/commands/pull.rs

```diff
@@ -386,22 +459,20 @@ impl Pull {
                             // TODO: this could probably benefit from some newlines
                             message::warning(display_chain(&err));
 
-                            let message_with_warning = formatdoc! {"
-                                {pulled_line}
+                            let message_with_warning = indoc! {"
+                                Migrated the environment to version 1 and included your system
+                                but could not build.
 
-                                Migrated the environment to version 1 and included your system but could not
-                                build.
```

Comment:
nonblocking: why this change? Shouldn't we be wrapping at 80?

Reply:
hmm i think that was the result of trying to make it look less weird, realizing its too wide and then applying semantic breaks.
I can undo this hunk